packages/portal/src/pages/account/index.vue

```diff
@@ -102,6 +102,7 @@
                           {{ $tc('items.itemCount', likedItems.length) }}
                         </h2>
                         <ItemSelectButton
+                          v-if="$features.itemMultiSelect"
                           class="ml-auto"
                           @select="(newState) => selectState = newState"
```

Comment:
Rename `selectState` to `itemMultiSelect` and provide from this page as well.

Reply:
Indeed. Done.